middleware/packet-forward-middleware/packetforward/types/params.go

```diff
@@ -0,0 +1,43 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// DefaultFeePercentage is the default value used to extract a fee from all forwarded packets.
+var DefaultFeePercentage = sdk.NewDec(0)
```

Comment:
```suggestion
var DefaultFeePercentage = sdkmath.LegacyNewDec(0)
```

Reply:
I would double check it works hahaha I place this suggestion of of memory but the point is to use the new math package with legacy Dec because this one will be deprecated